src/main/scala/org/virtuslab/yaml/internal/load/reader/YamlReader.scala

```diff
@@ -98,29 +102,37 @@ class YamlReader(in: CharSequence) extends Reader {
     val foldedIndent = indent
     skipUntilNextIndent(foldedIndent)
 
+    def chompedEmptyLines() =
+      while (peek() == Some('\n')) {
+        skipCharacter()
+        sb.append("\\n")
+      }
+
+      skipUntilNextIndent(foldedIndent)
```

Comment:
in [spec](https://yaml.org/spec/1.2/spec.html#id2794534) `chomping` _controls how final line breaks and trailing empty lines are interpreted_. For me `chomping` is also a very enigmatic word, I think that, for example, `eatNewlines` would be clearer